### ISSUE TITLE
Output Tests: Contains

### ISSUE THREAD

**Issue by joebell93** [oddbird/true#114]
It would be great to integrate the latest version of **true** in the [Sky Toolkit](https://github.com/sky-uk/toolkit) but one of the major things holding us back is the lack of a "**contains**"-like output test.

Not too dissimilar to "[Includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes)" in JavaScript
(but `@include includes {};` makes me feel [uncomfortable](https://pics.onsizzle.com/Instagram-208925.png))

Most of our UI mixins have a lot of inline documentation, it would be more efficient to test just the properties (or some of them) in this case, rather than copy/paste over the entire code-block over for the test to pass.

#### Example

For our [Panel Indicator](https://github.com/sky-uk/toolkit/blob/develop/packages/sky-toolkit-core/tools/_panels.scss#L24) mixin, we'd like to utilise a `@contains` mixin to check if an output _contains_ a set of properties, rather than a hard-match with all of our comments needing duplication in our tests.

```scss
@include test-module("@mixin panel-indicator") {
  @include test("should add panel indicator") {
    @include assert("sets default theme by default") {
      @include output() {
        @include panel-indicator();
      }

      @include contains() {
        background-color: #fff;
        box-shadow: inset 11px 11px 11px -11px #9f9f9f;
      }
    }

    @include assert("sets dark theme if specified") {
      @include output() {
        @include panel-indicator("dark");
      }

      @include contains() {
        background-color: #222;
        box-shadow: inset 11px 11px 11px -11px #000;
      }
    }
  }
}
```

I'd love to contribute back into **True** but I'm not too familiar with the codebase; if anyone's happy to steer me or advise on a solution I'd really appreciate it!

**Comment by mirisuzanne** [comment]
That's an interesting feature. 

On the Sass side, it can be a near clone of the `expect` mixin, with a small change in the generated comment-wrapper (https://github.com/oddbird/true/blob/master/sass/true/_assert.scss#L307 and line 317).

That will also require a change in the parser/mocha-comparison, which I'm less clear on. Maybe @carljm can speak to that.

**Comment by mirisuzanne** [comment]
Closed by #120 - Thanks @JakeChampion !

**Comment by joebell93** [comment]
Thank you @JakeChampion!
